### PR TITLE
Test Orka as privileged tasks

### PR DESCRIPTION
**Pull request by vinamra28**
Signed-off-by: vinamra28 <vinjain@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -6,14 +6,14 @@ set -e
 
 # Maximin number of parallel tasks run at the same time
 # # start from 0 so 4 => 5
-MAX_NUMBERS_OF_PARALLEL_TASKS=4
+MAX_NUMBERS_OF_PARALLEL_TASKS=1
```
Comment (chmouel) [inline openshift/e2e-tests-openshift.sh:9]: Is that what you want to do ? We had running as 0 before, and it was quite slow (would timeout the openshift-ci job), is 2 a good number ?

Comment (vinamra28) [reply openshift/e2e-tests-openshift.sh:9]: this is just for testing :sweat_smile: 
I have skipped `non-privileged` and just wanted to test the `privileged` ones. Also under privileged section I have put `orka-*`  tasks so testing 2 at a time.

Comment (vinamra28) [reply openshift/e2e-tests-openshift.sh:9]: with this --> `# test_non_privileged $(\ls -1 -d task/*/*/tests)` I guess all the non-privileged tasks would not be tested?

**Pull request closed without merging**